docs/pages/fragments/high-availability-k3s.mdx

```diff
@@ -0,0 +1,74 @@
+### Enabling High Availability
+
+
+In order to run vcluster with k3s as Kubernetes distribution in high availability mode, the following steps are required:
+
+* create and use an [external datastore](./external-datastore.mdx) (as opposed to the embedded SQLite datastore used in single-server setups)
+* create secret with a static value for the k3s server token (see the [official k3s FAQs](https://docs.k3s.io/faq#what-is-the-difference-between-k3s-server-and-agent-tokens) for more information)
+* run two or more k3s pods that will serve the Kubernetes API and run other control plane services
+
+First create a secret with a random value used as the k3s server token:
+
+```
+kubectl -n vcluster create secret generic my-k3s-tokens --from-literal=server-token=$(cat /dev/urandom | tr -dc A-Za-z0-9 | head -c32)
```

Comment:
Mhm, sounds good to me. Do you have any preference on how to generate the secret? With a Chart hook? With a named template and some [`lookup`](https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-lookup-function) magic?

Reply:
Hi @dakraus,
What I can suggest here is we give a `.Values.SecretToken` options for the user to supply. If its not supplied, 
we can generate a random token through helm and use that to create the secret.

```yaml
{{- define "chrt-6456.password" -}}
{{- randAlphaNum 24 | nospace -}}
{{- end -}}
```

You might want to be carefuly about where you define this template and how many times/where you are calling it.
Since we don't the random string to be generated again and again for different invocations, its advised to put relevant resources in the same template file. Ref: https://github.com/helm/helm-www/pull/877/files

Reply:
As a correction to the previous comment, to me it seems like we only need this token to be defined once for a release, hence we can do away with the complicacy of template and directly have it in our secret definitation 
```
data:
{{- if not .Values.SecretToken }}
  serverToken: {{ randAlphaNum 20 | b64enc }}
{{- else }}
  serverToken: {{ .Values.SecretToken }}
{{- end }}
```

Reply:
@ishankhare07 thanks for your suggestions. Yes, you are right that the token must be defined once for a release and I also added the possibility for the user to provide a custom value for the k3s server token.